mite_selenium/__init__.py

```diff
@@ -50,6 +50,7 @@ def _start(self):
             file_detector=self._file_detector,
             options=self._options,
         )
+        self._context.webdriver = self._remote
```

Comment:
I think that we ought to signal somehow that this should not be the first port of call for users, either by calling it `_webdriver` or `internal_webdriver` or something like that.  (I actually don't see a huge issue with the status quo where it can be accessed as `context.browser._remote`, although I can also see why that's not optimal...)

Reply:
I had initially gone the route of accessing the driver via `context.browser._remote`, but as you say, not optimal - Demeter and all that. Neither does it seem like an internal only attribute, but as you also point out, it shouldn't be the primary choice for accessing the Webdriver instance. How about naming the instance `raw_webdriver` or something similar to convey that secondary use intent?

Reply:
👍 for raw_webdriver